java/ql/src/semmle/code/java/frameworks/jackson/JacksonSerializability.qll

```diff
@@ -105,7 +126,7 @@ library class ExplicitlyReadJacksonDeserializableType extends JacksonDeserializa
 }
 
 /** A type used in a `JacksonDeserializableField` declaration. */
-library class FieldReferencedJacksonDeSerializableType extends JacksonDeserializableType {
+private class FieldReferencedJacksonDeSerializableType extends JacksonDeserializableType {
   FieldReferencedJacksonDeSerializableType() {
```

Comment:
Not directly related to your pull request, but could you please change the capitalization here for consistency?
```suggestion
private class FieldReferencedJacksonDeserializableType extends JacksonDeserializableType {
  FieldReferencedJacksonDeserializableType() {
```